projects/ad9361/src.mk

```
@@ -20,12 +20,19 @@ SRCS += $(DRIVERS)/axi_core/axi_adc_core/axi_adc_core.c			\
 	$(DRIVERS)/spi/spi.c						\
 	$(DRIVERS)/gpio/gpio.c						\
 	$(NO-OS)/util/util.c
-SRCS +=	$(PLATFORM_DRIVERS)/axi_io.c					\
-	$(PLATFORM_DRIVERS)/delay.c
+SRCS +=	$(PLATFORM_DRIVERS)/axi_io.c
+ifeq (linux,$(strip $(PLATFORM)))
+SRCS +=	$(PLATFORM_DRIVERS)/linux_delay.c
+SRCS +=	$(PLATFORM_DRIVERS)/linux_spi.c
+SRCS +=	$(PLATFORM_DRIVERS)/linux_gpio.c
+else
+SRCS +=	$(PLATFORM_DRIVERS)/delay.c
+endif
```

Maybe this is simpler. Same for .h files
```suggestion
SRCS +=	$(PLATFORM_DRIVERS)/$(PLATFORM)_spi.c
SRCS +=	$(PLATFORM_DRIVERS)/$(PLATFORM)_gpio.c
```
and: 
```
ifeq (linux,$(strip $(PLATFORM)))
SRCS +=	$(PLATFORM_DRIVERS)/linux_delay.c
else
SRCS +=	$(PLATFORM_DRIVERS)/delay.c
endif
```
Or rename linux_delay.c -> delay.c

---

Done it for $(PLATFORM)_spi.c and $(PLATFORM)_gpio.c. For consistency, I would better rename delay.c to $(PLATFORM)_delay.c.